code/modules/surgery/surgery.dm

```diff
@@ -86,10 +86,14 @@
 
 	var/datum/surgery_step/S = get_surgery_step()
 	if(S)
+		var/obj/item/tool = user.get_active_held_item()
 		if(S.try_op(user, target, user.zone_selected, user.get_active_held_item(), src, try_to_fail))
 			return TRUE
 		if(iscyborg(user) && user.a_intent != INTENT_HARM) //to save asimov borgs a LOT of heartache
 			return TRUE
+		if(tool.item_flags & SURGICAL_TOOL) //Just because you used the wrong tool it doesn't mean you meant to whack the patient with it
```

Comment:
//Tool commonly used for surgery: won't attack targets in an active surgical operation on help intent (in case of mistakes)

Doesn't actually check for help intent. Intentional?

Reply:
Intentional, it's already done before the SURGICAL_TOOL check so it would be pointless to repeat it